chia/full_node/weight_proof.py

```diff
@@ -1533,18 +1532,18 @@ def _get_last_ses_hash(
 ) -> tuple[Optional[bytes32], uint32]:
     for idx, block in enumerate(reversed(recent_reward_chain)):
         if (block.reward_chain_block.height % constants.SUB_EPOCH_BLOCKS) == 0:
-            idx = len(recent_reward_chain) - 1 - idx  # reverse
+            reversed_idx = len(recent_reward_chain) - 1 - idx  # reverse
```

Comment:
etc...  i think this is more fitting given that `idx` is itself reversed effectively.  `idx` can't even be used to index anything per se.

```suggestion
            original_idx = len(recent_reward_chain) - 1 - idx  # reverse
```

Reply:
Taken.  Gonna ping @almogdepaz for this section too I think.